packages/tocco-ui/src/Button/Button.stories.js

```diff
@@ -41,61 +41,62 @@ storiesOf('Button', module)
     () => {
       return [
         <Button
+          dense={boolean('dense', false) || undefined}
```

Comment:
```
const knobs = {dense: boolean('dense', false) || undefined, disabled: boolean('disabled', false) || undefined}

<Button
  ...knobs
  key="1"
```

So you dont have to repeat yourself that often

Reply:
see 88008f1